src/main/java/org/folio/dew/repository/SFTPObjectStorageRepository.java

```diff
@@ -86,21 +86,21 @@ public boolean upload(String username, String password, String host, int port, S
       return true;
     } catch (Exception e) {
       log.info("Error uploading the file", e);
-      throw new EdifactException(String.format("Unable to upload to %s:%d%s. %s", host, port, folder, e.getMessage()));
+      throw new EdifactException(String.format("Unable to upload to %s:%d/%s. %s", host, port, folder, e.getMessage()));
     }
   }
 
   public byte[] download(SftpClient sftpClient, String path) {
-    byte[] fileBytes = null;
     try {
       InputStream stream = sftpClient.read(path);
       log.info("File found from path: {}", path);
-      fileBytes = stream.readAllBytes();
+      byte[] fileBytes = stream.readAllBytes();
       stream.close();
```

Comment:
in already existed code if exception will be thrown - stream will not be closed, lets improve this by using finally block or try with resources